src/components/Radio/Radio.tsx

```diff
@@ -0,0 +1,101 @@
+/** @jsxRuntime classic /
+/** @jsx jsx */
+import { jsx } from "theme-ui"
+import { ChangeEvent } from "react"
+import useRadioGroup from "../RadioGroup/hooks/useRadioGroup"
+import { RadioGroup } from "../RadioGroup/RadioGroup"
+export interface RadioProps {
+  /** The label content */
+  label?: string
+  /** The value of the `input` element, required for a controlled component */
+  value?: string
+  /** Callback fired when the value is changed */
+  onChange?: (event: ChangeEvent<HTMLInputElement>) => void
+  /** If `true`, the `input` is required */
+  required?: boolean
+  /** The id of the `input` element. Use this prop to make label and `helperText` accessible for screen readers */
+  id?: string
+  /** Radio identification */
+  name?: string
+}
+
+const Radio = ({
+  id,
+  value,
+  label,
+  name,
+  onChange,
+  required = false,
+}: RadioProps) => {
+  const radioGroup = useRadioGroup()
+
+  if (radioGroup) {
+    if (typeof onChange === "undefined") {
+      onChange = radioGroup.onChange
+    }
+    if (typeof name === "undefined") {
+      name = radioGroup.name
+    }
+  }
```

Comment:
Shouldn't it also use the `value` from the radioGroup context, when it's a controlled radio group?

Reply:
done! ✅